src/core/index.js

```diff
@@ -133,6 +133,15 @@ class IPFS extends EventEmitter {
       isIPFS: isIPFS
     }
 
+    // ipfs.files
+    const mfs = components.mfs(this)
+
+    Object.keys(mfs).forEach(key => {
+      if (mfs.hasOwnProperty(key)) {
```

Comment:
Minor: I believe `Object.keys` iterates over own properties so I think this check is redundant, unless I'm missing something?

Reply:
From my understanding `Object.keys` returns all properties, including inherited ones whereas `hasOwnProperty` will only return true if it's a non-inherited property.  Happy to change this if you prefer it the other way.